package.json

```diff
@@ -64,6 +65,11 @@
 				"command": "dreammaker.findReferencesTree",
 				"title": "Find All References (Object Tree)",
 				"category": "DreamMaker"
+			},
+			{
+				"command": "dreammaker.returnDreamdaemonPath",
```

Comment:
As far as I can tell, the command doesn't need to be registered here to be useable in task definitions as `${command:dreammaker.returnDreamdaemonPath`, so this chunk can be left out